gatsby-site/gatsby-node.js

```diff
@@ -42,8 +42,8 @@ const { createRemoteFileNode } = require("gatsby-source-filesystem")
  * @property {string | null} twitter
  * @property {string | null} signup_link
```

Comment:
do we need to check for undefined for these fields too?

Reply:
We do not, it seems that airtable only does this with the array fields